modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/dto/v1_0/converter/ObjectEntryDTOConverter.java

```diff
@@ -300,14 +291,33 @@ else if ((nestedFieldsDepth > 0) &&
 							false
 						)) {
 
-						map.put(
-							StringUtil.replaceLast(objectFieldName, "Id", ""),
-							_toDTO(
-								_getDTOConverterContext(
-									dtoConverterContext, objectEntryId),
-								nestedFieldsDepth - 1,
-								_objectEntryLocalService.getObjectEntry(
-									objectEntryId)));
+						ObjectRelationship objectRelationship =
+							_objectRelationshipLocalService.
+								fetchObjectRelationshipByObjectFieldId2(
+									objectField.getObjectFieldId());
+
+						ObjectDefinition objectDefinition1 =
```

Comment:
Since there is no variable objectDefinition2, we don't need to add the 1 at the end of the variable objectDefinition1. Could you please remove the 1 from the name?

Reply:
I changed it to `objectDefinitionRelated` because the method signature already had a variable named `objectDefinition`.
What do you think about this rename?